GlyssenEngineTests/Character/CharacterAssignerTests.cs

```diff
@@ -1,8 +1,12 @@
-﻿using System;
+﻿/// <summary>
```

Comment:
Remove this comment

Reply:
My last change. I can't make it easy for you can I?  Sorry.  Done.